### DISCUSSION TITLE
Customization of email sent by self hosted docker plausible.

### DISCUSSION THREAD

**Discussion by madanyang** [plausible/analytics#2684] [Self-Hosted Support] [answered]
Well as the title says, I need to customize the emails sent out by the self hosted plausible. The mails have content mentioning plausible.io but that is not what I want to see.  Same is true for the website pages. I would prefer that it looks not as plausible.io but similar to my other sites, logo,fonts footers etc.

1. How and where can I change the mails content, logo etc. 
2. How and where can I change the website footers and navigation.

Thanks in advance

**Comment by ruslandoga** [comment, marked as answer]
👋 @madanyang 

These changes cannot be done via configuration right now, so you'd need to do them in code and rebuild the container image.

1. For emails you can change the content of the [templates](https://github.com/plausible/analytics/tree/master/lib/plausible_web/templates/email) and [the base email;](https://github.com/plausible/analytics/blob/master/lib/plausible_web/templates/layout/base_email.html.eex) the logo can be changed [here](https://github.com/plausible/analytics/blob/535874be6f83bf158752e757b25291d42321ff2e/lib/plausible_web/templates/layout/_header.html.eex#L7-L8)
2. [Footers are here](https://github.com/plausible/analytics/blob/master/lib/plausible_web/templates/layout/_footer.html.eex) and I'm not sure what you mean by navigation

**Reply by madanyang** [reply]
Thanks for the info,

It could be a nice feature to include maybe in the future, if there would be a possibility to define items that could be changed and use them with templates, then those could be changed via the environment variables. That would certainly remove the rebuilding the docker files.
